src/Uno.UWP/Devices/Sensors/Magnetometer.iOS.cs

```diff
@@ -0,0 +1,63 @@
+﻿#if __IOS__
```

Comment:
really we should set up Uno.UWP to recognise `*.ios.cs` etc (but it's not necessary for this PR, just thinking out loud)